src/xdocs/config_filters.xml

```diff
@@ -110,11 +110,8 @@
             file is edited.
           </p>
           <p>
-            Usage: This filter only works in conjunction with a <code>FileContentsHolder</code>, since that check makes
```

Comment:
nobody will read this note till it fail their build :) , nobody read releasenotes before update, usually user try to update, if some problems happen start to read release notes and then review documentation.
So we will update it on moment when it become affecting users.